Makefile

```diff
@@ -11,9 +11,9 @@ PREFIX  := installation-images-$(VERSION)
 BUILD_ID := $(shell [ -f .build_id ] || bin/build_id > .build_id ; cat .build_id)
 
 # build initrd+modules+gefrickel after tftp (it needs the sha256 sums over the other images)
-COMMON_TARGETS	     := rescue root root+rescue bind libstoragemgmt gdb mini-iso-rmlist tftp initrd+modules+gefrickel
+COMMON_TARGETS	     := rescue root root+rescue bind libstoragemgmt gdb libyui-rest-api mini-iso-rmlist tftp initrd+modules+gefrickel
 # keep in sync with data/boot/tftp.file_list
```

Comment:
You need to add the image to `tftp.file_list` else it won't be included in the `tftpboot-installation-xxx` packages.